apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/PaginationTest.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+
+namespace Google.Cloud.Compute.V1.IntegrationTests
+{
+    /// <summary>
+    /// Integration tests focused on pagination
+    /// </summary>
+    [Collection(nameof(ComputeFixture))]
+    public sealed class PaginationTest
+    {
+        private readonly ComputeFixture _fixture;
+
+        public PaginationTest(ComputeFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public void ListAcTypes()
```

Comment:
Note to self: rename this to ListAcceleratorTypes. (The abbreviation is unnecessary and makeds it harder to understand.)